content/20.pharmaceuticals.md

```diff
@@ -127,21 +128,51 @@ Therefore, even though the study reported clinical improvement in 68% of the 53
+Those who received remdesivir had a median recovery time of 10 days, as compared with 15 days in those who received placebo (rate ratio for recovery, 1.29; 95% CI, 1.12 to 1.49; P<0.001).
+The Kaplan-Meier estimates of mortality by 14 days were 6.7% with remdesivir and 11.9% with placebo, with a hazard ratio (HR) for death of 0.55 and a 95% CI of 0.36 to 0.83, and at day 29, remdesivir corresponded to 11.4% and the placebo to 15.2% (HR: 0.73; 95% CI: 0.52 to 1.03).
+Serious adverse events were reported in 131 of the 532 patients who received remdesivir (24.6%) and in 163 of the 516 patients in the placebo group (31.6%).
+This study also reported an association between remdesivir administration and both clinical improvement and a lack of progression to more invasive respiratory intervention in patients receiving non-invasive and invasive ventilation at randomization [@doi:10.1056/NEJMoa2007764].
 Largely on the results of this trial, the FDA issued an Emergency Use Authorization (EUA) for remdesivir for the treatment of hospitalized COVID-19 patients.
 Clinical trials are currently underway to evaluate the use of remdesivir to treat COVID-19 patients at both early and late stages of infection and in combinations with other drugs (Figure @fig:ebm-trials).
 These trials include [@doi:10.1038/s41422-020-0282-0; @clinicaltrials:NCT04292730; @url:https://www.clinicaltrialsregister.eu/ctr-search/trial/2020-000936-23/FR; @clinicaltrials:NCT04252664; @clinicaltrials:NCT04257656].
 As of October 22, 2020, remdesivir received FDA approval based on three clinical trials [@url:https://www.fda.gov/news-events/press-announcements/fda-approves-first-treatment-covid-19].
 
+However, results suggesting no effect of remdesivir on survival were reported by the WHO Solidarity trial [@doi:10.1056/NEJMoa2023184].
+This large-scale, open-label trial enrolled 11,330 adult in-patients at 405 hospitals in 30 countries around the world [@doi:10.1056/NEJMoa2023184].
+Patients were randomized in equal proportions into four experimental and a control conditions, corresponding to four candidate treatments for COVID-19 and SOC, respectively; no placebo was administered.
+The 2750 patients in the remdesivir group were administered 200 mg intravenously on the first day and 100 mg on each subsequent day until day 10 and assess for in-hospital death (primary endpoint), duration of hospitalization, and progression to mechanical ventilation.
+There were also 2708 control patients who would have been eligible and able to receive remdesivir were they not assigned to the control group.
+A total of 604 patients among these two cohorts deceased during initial hospitalization.
+The rate ratio of death between these two groups was therefore not significant (p = 0.50), suggesting that the administration of remdesivir did not affect survival.
```

Comment:
```suggestion
The rate ratio of death between these two groups was therefore not significant (_p_ = 0.50), suggesting that the administration of remdesivir did not affect survival.
```

Reply:
I’ll let you make a decision on what way to write the p values for consistency - @rando2